utils/s2n_random.c

```diff
@@ -568,8 +568,11 @@ S2N_RESULT s2n_set_private_drbg_for_test(struct s2n_drbg drbg)
 /*
  * volatile is important to prevent the compiler from
  * re-ordering or optimizing the use of RDRAND.
+ *
+ * This is marked with ASAN_IGNORE because address sanitizer is unable to deal
+ * with the inline assembly and emits false positives.
```

Comment:
It's not a false positive, it's an actual bug
https://github.com/aws/s2n-tls/pull/4310